_includes/swc/setup.html

```diff
@@ -94,12 +94,16 @@ <h3>The Bash Shell</h3>
           See the Git installation <a href="https://www.youtube.com/watch?v=9LQhwETCdwY ">video tutorial</a>
           for an example on how to open the Terminal.
           You may want to keep Terminal in your dock for this workshop.
-	</p>
-	<p>
-          To see if your default shell is Bash type <code>echo $SHELL</code> in
-          Terminal and press the enter/return key.  If the message printed does
-	  not end with '/bash' then your default is something else and you can run
-	  Bash by typing <code>bash</code>.
+        </p>
+        <p>
+            In Terminal, check if you have bash. To see if your default shell is Bash
```

Comment:
```suggestion
             To see if your default shell is Bash
```
Unless the above is untrue Bash is always there, the checking step is only to see if it's the default. 

If it's not always there, then we could add the step to check if it is available in addition to if it's default.